modules/repository/variables.tf

```diff
@@ -75,8 +75,7 @@ variable "has_downloads" {
 }
 
 locals {
-  default_topics = ["stakater", "chart", "${lower(replace(replace(var.name, "_", "-"), ".", "-"))}"]
-  topics         = "${concat(local.default_topics, var.topics)}"
+  topics = ["${compact(split(",", lower(replace(replace(replace(join(",", var.topics), " ", "-"), "_", "-"), ".", "-"))))}"]
```

Comment:
@faizanahmad055 let's use regex here instead of nested replace.
I think the format may be as follows:
`topics = ["${compact(split(",", lower(replace(join(",", var.topics), "/\s|_|\./", "-"))))}"]`

Reply:
Updated